packages/mongodb/src/resolvers.ts

```diff
@@ -19,6 +19,10 @@ export async function resolveQueryObjectId(
 ): Promise<IdQueryObject<ObjectId>>
 export async function resolveQueryObjectId(value: ObjectIdParam): Promise<ObjectId>
 export async function resolveQueryObjectId(value: ObjectIdParam | IdQueryObject<ObjectIdParam>) {
+  if (value === undefined || value === null) {
```

Comment:
I think this is a scenario where we can actually just do `if (!value)` since the value has to be a string or an object, and neither of those can be `0`.

Reply:
I think this is ok. Otherwise it would still error if you pass `null` as the value.

Reply:
Oops, I already made the changes prior to reading this. Resubmitted.